stylelint.config.js

```diff
@@ -1,3 +1,6 @@
 const stylelintConfig = require('@nextcloud/stylelint-config')
 
+stylelintConfig.rules['at-rule-no-unknown'] = null
+stylelintConfig.rules['scss/at-rule-no-unknown'] = true
```

Comment:
I'm not fully sure why the at-rule-no-unknown rule takes precedence over the scss rule but maybe this is something to be added to https://github.com/nextcloud/stylelint-config/blob/master/index.js

Reply:
https://github.com/stylelint-scss/stylelint-scss/blob/c167e8cfcd05d511b00b71b848dee3a46ddd3147/src/rules/at-rule-no-unknown/README.md

Reply:
Should be pulled in from https://github.com/stylelint-scss/stylelint-config-recommended-scss/blob/master/index.js#L8 :shrug: